DESCRIPTION

```diff
@@ -53,7 +53,12 @@ Imports:
     joyn,
     dm,
     config,
-    collapse
+    collapse,
```

Comment:
Do we need all these dependencies? For example, do we need `covr`? Could it be moved to `suggests`?

Reply:
I moved covr package to suggests section.